README.md

```diff
@@ -257,7 +267,7 @@ Come with us to make an awesome *Buchu*.
 
 Now, if you do not have technical knowledge and also have intend to help us, do not feel shy, [click here](https://github.com/herbsjs/buchu/issues) to open an issue and collaborate their ideas, the contribution may be a criticism or a compliment (why not?)
 
-If you would like to help contribute to this repository, please see [CONTRIBUTING](https://github.com/herbsjs/buchu/blob/master/.github/CONTRIBUTING.md)
+We have some conventions to contribute to the *Buchu* project, see more information in our [CONTRIBUTING.md](CONTRIBUTING.md). So please, read this before send to us a [pull requests](https://github.com/herbsjs/buchu/pulls).
```

Comment:
I believe that this should not be here. The documentation has been updated in this part.